.github/workflows/ci.yml

```diff
@@ -158,7 +158,7 @@ jobs:
         with:
           activate-environment: sunpy-test
           environment-file: sunpy-dev-env.yml
-          python-version: 3.10
+          python-version: 3.11
```

Comment:
Dont think will actually change anything.

Reply:
shouldn't this be a string? i.e. `3.10 != "3.10"`?